pkg/controller/controller.go

```diff
@@ -228,22 +231,42 @@ func startCloudControllerManager(ctx context.Context, clusterName string, kubeCl
 	ctx, cancel := context.WithCancel(ctx)
 	go serviceController.Run(ctx, 5, ccmMetrics)
 
-	// Start the node controller
-	nodeController, err := nodecontroller.NewCloudNodeController(
-		sharedInformers.Core().V1().Nodes(),
-		kubeClient,
-		cloud,
-		30*time.Second,
-		5, // workers
-	)
+	clustersInterface, ok := cloud.Clusters()
+	if !ok {
+		klog.Fatalf("Unable to use clusters interface: %v", err)
+	}
+
+	masterName, err := clustersInterface.Master(ctx, clusterName)
```

Comment:
This seems overly complicated to get the masterName. I could change it to:
```
masterName := clusterName+'-control-plane'
```

Thoughts?